### PR TITLE
Bug 1105330 - Footer logo cut

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

**Comment by tsl143** [comment]
@spasovski , is this good to go?

File: src/media/css/base.styl [hunk 1]
```diff
@@ -9,6 +9,7 @@ to this project.
 }
 #footzilla {
     background-image: url(../img/footer.png);
+    background-size: auto 22px;
```
Comment (spasovski) [inline src/media/css/base.styl]: Thanks, you're on the right track here but IE and Safari have issues with `background-size: auto` so we should make this `background-size: 81px 21px` and have it apply to both regular and high pixel density screens.

**Comment by tsl143** [comment]
Updated !

**Comment by spasovski** [comment]
Thanks r+

**Pull request merged**